iac/timeclock-dashboard/aws_s3_bucket.website.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_s3_bucket" "website" {
+    bucket = "timeclock-dashboard"
```

Comment:
For names of resources, you'll want to do something like `"${local.hidashNamePrefix}-timeclock-dashboard"`. This way it will be named to the environment.

AWS does not make it clear what account your in/ it is easy to forget. Naming resources makes it obvious. Also, we only have one account since only one account in an org will get the free tier so both environments are in the same account.